paimon-core/src/main/java/org/apache/paimon/table/sink/TableCommitImpl.java

```diff
@@ -129,6 +131,12 @@ public TableCommitImpl ignoreEmptyCommit(boolean ignoreEmptyCommit) {
         return this;
     }
 
+    @Override
+    public InnerTableCommit withMetricRegistry(AbstractMetricRegistry registry) {
+        commit.withMetrics(new CommitMetrics(registry, tableName));
+        return this;
+    }
+
```

Comment:
I didn't find where to call this method `withMetricRegistry`

Reply:
This method is called from Flink / Spark operator. We'll introduce a `FlinkMetricRegistry` in the next PR.